public/app/features/alerting/unified/components/rule-editor/alert-rule-form/simplifiedRouting/contactPoint/ContactPointDetails.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { ReactNode } from 'react';
+
+import { Icon, Stack, Text } from '@grafana/ui';
+import { INTEGRATION_ICONS } from 'app/features/alerting/unified/types/contact-points';
+import { GrafanaNotifierType } from 'app/types';
+
+import { RECEIVER_META_KEY, RECEIVER_PLUGIN_META_KEY } from '../../../../contact-points/useContactPoints';
+import { ReceiverConfigWithMetadata, getReceiverDescription } from '../../../../contact-points/utils';
+import { ReceiverMetadataBadge } from '../../../../receivers/grafanaAppReceivers/ReceiverMetadataBadge';
+import { ReceiverPluginMetadata } from '../../../../receivers/grafanaAppReceivers/useReceiversMetadata';
+
+interface ContactPointDetailsProps {
+  receivers: ReceiverConfigWithMetadata[];
+}
+
+export const ContactPointDetails = ({ receivers }: ContactPointDetailsProps) => {
+  return (
+    <Stack direction="column" gap={0}>
+      <div>
+        {receivers.map((receiver, index) => {
+          const metadata = receiver[RECEIVER_META_KEY];
+          const pluginMetadata = receiver[RECEIVER_PLUGIN_META_KEY];
+          const key = metadata.name + index;
+          return (
+            <ContactPointReceiver
+              key={key}
+              name={metadata.name}
+              type={receiver.type}
+              description={getReceiverDescription(receiver)}
+              pluginMetadata={pluginMetadata}
+            />
+          );
+        })}
+      </div>
+    </Stack>
+  );
+};
+
+interface ContactPointReceiverProps {
+  name: string;
+  type: GrafanaNotifierType | string;
+  description?: ReactNode;
+  pluginMetadata?: ReceiverPluginMetadata;
+}
+
+const ContactPointReceiver = (props: ContactPointReceiverProps) => {
```

Comment:
I couldn't re-use the existing `ContactPointReceiver` as was rendering and requiring more information and props than we need in this case.

Reply:
We could split up the existing `ContactPointReceiver` in to separate components, we already have one for the metadata with `ContactPointReceiverMetadataRow` so maybe something like `ContactPointReceiverTitleRow`?

Reply:
💯